### PR TITLE
CAD-517 | chainsync nightly:  fix

### PR DESCRIPTION
**Pull request by deepfire**
Fix the nightly chainsync benchmark -- convert the affected pieces to structured logging.

### CODE REVIEW

File: benchmarking/chain-sync/configuration/log-config-ci.yaml [hunk 1]
```diff
@@ -20,11 +20,16 @@ setupScribes:
   - scKind: FileSK
     scName: "node-0.log"
     scFormat: ScText
```
Comment (CodiePP) [inline benchmarking/chain-sync/configuration/log-config-ci.yaml]: we can remove the "node-0.log" textual output.

File: benchmarking/chain-sync/analyse-logs.sh [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-LASTNETIN=`grep -e '.*cardano.node-metrics:.*Net.IpExt:InOctets =' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*Net.IpExt:InOctets = \([0-9]\+\).*/"\1 \2";\3/p;' | sort -k 1.2,1.23 | tail -n 1`
-LASTNETOUT=`grep -e '.*cardano.node-metrics:.*Net.IpExt:OutOctets =' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*Net.IpExt:OutOctets = \([0-9]\+\).*/"\1 \2";\3/p;' | sort -k 1.2,1.23 | tail -n 1`
-LASTDISKIN=`grep -e '.*cardano.node-metrics:.*IO.rbytes =' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*IO.rbytes = \([0-9]\+\).*/"\1 \2";\3/p;' | sort -k 1.2,1.23 | tail -n 1`
-LASTDISKOUT=`grep -e '.*cardano.node-metrics:.*IO.wbytes =' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*IO.wbytes = \([0-9]\+\).*/"\1 \2";\3/p;' | sort -k 1.2,1.23 | tail -n 1`
+LOGFILE_JSON=`ls -1r state-node-${NETWORK}/node-0-*.json | head -2`
+echo $LOGFILE_TEXT
+LASTSLOT=`jq 'select (.data.event.kind == "TraceCopyToImmDBEvent.CopiedBlockToImmDB") | "\(.at);\(.data.tip | sub (".*@(?<slot>.*)$"; "\(.slot)"))"' $LOGFILE_JSON | sed 's/T/ /; s/\(20[0-9][0-9]\)/"\1/; s/Z/"/' | sort -k 1.2,1.23 | tail -n 1`
```
Comment (CodiePP) [inline benchmarking/chain-sync/analyse-logs.sh]: this does not yet work.

File: benchmarking/chain-sync/analyse-logs.sh [hunk 3]
```diff
@@ -3,17 +3,17 @@
 NETWORK=${1:-mainnet}
 
 # get first block copy time
-LOGFILE=`ls -1 state-node-${NETWORK}/node-0-*.log | head -3`
-FIRSTSLOT=`grep -e '.*:cardano.node.ChainDB:.*TraceCopyToImmDBEvent.CopiedBlockToImmDB.*"tip":"[a-z0-9]*@' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*"tip":"[a-z0-9]*@\([0-9]\+\)".*/"\1 \2";\3/p; q;'`
+LOGFILE_JSON=`ls -1 state-node-${NETWORK}/node-0-*.json | head -2`
+FIRSTSLOT=`jq --slurp 'map (select (.data.event.kind == "TraceCopyToImmDBEvent.CopiedBlockToImmDB")) | .[0] | "\(.at);\(.data.tip | sub (".*@(?<slot>.*)$"; "\(.slot)"))"' $LOGFILE_JSON | xargs echo | sed 's/\([0-9]\)T\([0-9]\)/\1 \2/; s/\(20[0-9][0-9]\)/"\1/; s/Z/"/'`
```
Comment (CodiePP) [inline benchmarking/chain-sync/analyse-logs.sh]: this does not yet work.

File: benchmarking/chain-sync/analyse-logs.sh (hunk 2 above)
Comment (deepfire) [reply benchmarking/chain-sync/analyse-logs.sh]: Fixed.

File: benchmarking/chain-sync/analyse-logs.sh (hunk 3 above)
Comment (deepfire) [reply benchmarking/chain-sync/analyse-logs.sh]: Hmm, it did work locally, interesting.

Comment (deepfire) [reply benchmarking/chain-sync/analyse-logs.sh]: @CodiePP, where were you seeing it as not working -- it did both locally and on CI: https://buildkite.com/input-output-hk/cardano-node-nightly-sync/builds/26#0cd7b168-25cd-498e-a957-a62dde1222af

File: benchmarking/chain-sync/configuration/log-config-ci.yaml (hunk 1 above)
Comment (deepfire) [reply benchmarking/chain-sync/configuration/log-config-ci.yaml]: @CodiePP, that requires converting the rest to the structured logging -- will do.

Comment (deepfire) [reply benchmarking/chain-sync/configuration/log-config-ci.yaml]: Done, pushed conversion to the structured logs.

Comment (CodiePP) [reply benchmarking/chain-sync/configuration/log-config-ci.yaml]: you just have overwritten my commit which allowed me to remove the node-0.log textual log.
I will force push again...

**Comment by CodiePP** [comment]
closing in favor of #557 which builds on structured logging and selects the correct fields.

**Pull request closed without merging**